### ISSUE TITLE
Default workflow download and default model.

### ISSUE THREAD

**Issue by comfyanonymous** [Comfy-Org/ComfyUI_frontend#451]
https://github.com/Comfy-Org/ComfyUI_frontend/blob/9e3dffd7fdcb855f997873d7d2c1361cd5df4284/src/scripts/defaultGraph.ts#L140

Having a full fp32 checkpoint is wasteful and we should be defaulting to a better model than SD1.5.

**Comment by robinjhuang** [comment]
@comfyanonymous What do you suggest? SDXL?

**Comment by huchenlei** [comment]
> @comfyanonymous What do you suggest? SDXL?

He means that we should have a link to download fp16 SD1.5 checkpoint. The current link downloads a fp32 checkpoint, which is twice the size.

**Comment by robinjhuang** [comment]
@huchenlei for sure. The second part of what he said was he need to default to a better model though?

**Comment by JorgeR81** [comment]
I suggest:

**_Juggernaut XL_**
_( The creator now works with RunDiffusion )_
https://huggingface.co/RunDiffusion/Juggernaut-X-v10
https://civitai.com/models/133005/juggernaut-xl

Alternatively:

**_RealVisXL_**
https://civitai.com/models/139562/realvisxl-v40
https://huggingface.co/SG161222

**_Dreamshaper XL Lightning_** 
_( There is no regular version, for the latest release )_
https://huggingface.co/Lykon/dreamshaper-xl-lightning
https://civitai.com/models/112902?modelVersionId=354657

**Comment by JorgeR81** [comment]
About Juggernaut XL,

They also have a detailed prompt guide for version 10 
https://learn.rundiffusion.com/prompting-guide-for-juggernaut-x/


Version 11 does not seem to be downloadable yet, without sign in to huggingface 
https://huggingface.co/RunDiffusion/Juggernaut-XI-v11

It's not in civitai yet.
They plan to release v11 there, once v12 is available via API.
Here is the latest comment on civitai, from the creator, about v11.

![civit](https://github.com/user-attachments/assets/18f46359-a34f-4cb2-bd38-f5272a372eb8)

**Comment by JorgeR81** [comment]
If you still want SD15,

Go for Realistic Vision V5.1 ( with VAE ) instead of V6.0 B1
https://civitai.com/models/4201?modelVersionId=130072

The latest version is V5.1 Hyper, ( released after V6.0 B1 ), in case you want to use a "turbo" version.
https://civitai.com/models/4201?modelVersionId=501240

---

But the Realistic Vision models, in huggingface, are all without VAE.
https://huggingface.co/SG161222

**Comment by marduk191** [comment]
why worry about such trivial things when half the stuff in main is broken due to not testing a single time?

**Comment by mcmonkey4eva** [comment]
The default model should not be changed to a random finetune, it should always definitely start as a basic core foundational model (ie one with no particular bias in any direction). Given that large models like Flux might be an issue for some reasons with weaker harder, I think SD 1.5 and SDXL 1.0 Base are the two available choices between the two at current time (pending something like an OMI model release eventually), so SDXL 1.0 Base would be the ideal modern default workflow.

If we're keeping to SD1.5, we could upload a pruned SD1.5 fp16 safetensors to huggingface/comfy-org (as the original official huggingface one doesn't have fp16 at all).

**Comment by JorgeR81** [comment]
Maybe you could have some sort of starting menu, in case no model is detected, where new users could select the model they want to download, from a curated list, including finetunes and base models.
This way you don't have to "endorse" a specific finetune.

Keep in mind that the inexperienced users that will actually be downloading the "preselected" model, will not be expecting such an outdated quality standard.
Any casual user that has generated an image online will have, by now, used a better model.
These users may not even notice that they are using an "historical" model version in Comfy UI.
They may end up associating that low quality with Comfy UI itself, and move on to other alternatives.

**Comment by robinjhuang** [comment]
> The default model should not be changed to a random finetune, it should always definitely start as a basic core foundational model (ie one with no particular bias in any direction). Given that large models like Flux might be an issue for some reasons with weaker harder, I think SD 1.5 and SDXL 1.0 Base are the two available choices between the two at current time (pending something like an OMI model release eventually), so SDXL 1.0 Base would be the ideal modern default workflow.
> 
> If we're keeping to SD1.5, we could upload a pruned SD1.5 fp16 safetensors to huggingface/comfy-org (as the original official huggingface one doesn't have fp16 at all).

This sounds good to me! I can take care of this.

**Comment by JorgeR81** [comment]
Yeah, but at least update the "readme" somewhere explaining that this was a foundational model, released in 2023, recommended for lower spec systems, and may not have the same quality as newer finetunes of the model.
